### PR TITLE
DIG.PWAutoLogin version 0.2.1

### PR DESCRIPTION
**Pull request by DIG-**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/119247)

### CODE REVIEW

**Comment by DIG-** [comment]
@microsoft-github-policy-service agree [company="DIG"]

**Comment by DIG-** [comment]
@microsoft-github-policy-service agree company="DIG"

**Comment by stephengillie** [comment]
`x86` Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Hi @DIG-,

Is there an installer switch to select installer scope?

**Comment by DIG-** [comment]
Hello @stephengillie,
Thanks for the report. I was thinking that the problem was that the installer isn't signed.
But since it's the installer dialog option then it has solution.
As a inno setup installer, there is `/ALLUSERS` or `/CURRENTUSER`.
I would suggest to use `/CURRENTUSER`.
If I make a new commit will be the validations triggered again?

**Comment by stephengillie** [comment]
Hi @DIG-,

We should probably add both, with `machine` and `user` scopes respectively.

File: manifests/d/DIG/PWAutoLogin/0.2.1/DIG.PWAutoLogin.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: DIG.PWAutoLogin
+PackageVersion: 0.2.1
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/DIG-/pw-auto-login/releases/download/0.2.1/pwal-setup.exe
+  InstallerSha256: 3ADF81DD57F314E342D276128BA585EA251458FB2E2DDB7EEB76FB29CA78BA9F
+- Architecture: x86
+  InstallerUrl: https://github.com/DIG-/pw-auto-login/releases/download/0.2.1/pwal-setup.exe
+  InstallerSha256: 3ADF81DD57F314E342D276128BA585EA251458FB2E2DDB7EEB76FB29CA78BA9F
```
Comment (stephengillie) [inline manifests/d/DIG/PWAutoLogin/0.2.1/DIG.PWAutoLogin.installer.yaml:13]:
```suggestion
- Architecture: x64
  Scope: machine
  InstallerUrl: https://github.com/DIG-/pw-auto-login/releases/download/0.2.1/pwal-setup.exe
  InstallerSha256: 3ADF81DD57F314E342D276128BA585EA251458FB2E2DDB7EEB76FB29CA78BA9F
  InstallerSwitches:
    Custom: /AllUsers
- Architecture: x64
  Scope: user
  InstallerUrl: https://github.com/DIG-/pw-auto-login/releases/download/0.2.1/pwal-setup.exe
  InstallerSha256: 3ADF81DD57F314E342D276128BA585EA251458FB2E2DDB7EEB76FB29CA78BA9F
  InstallerSwitches:
    Custom: /CurrentUser
- Architecture: x86
  Scope: machine
  InstallerUrl: https://github.com/DIG-/pw-auto-login/releases/download/0.2.1/pwal-setup.exe
  InstallerSha256: 3ADF81DD57F314E342D276128BA585EA251458FB2E2DDB7EEB76FB29CA78BA9F
  InstallerSwitches:
    Custom: /AllUsers
- Architecture: x86
  Scope: user
  InstallerUrl: https://github.com/DIG-/pw-auto-login/releases/download/0.2.1/pwal-setup.exe
  InstallerSha256: 3ADF81DD57F314E342D276128BA585EA251458FB2E2DDB7EEB76FB29CA78BA9F
  InstallerSwitches:
    Custom: /CurrentUser
```

**Comment by DIG-** [comment]
I checked with a virtual machine and it shows the SmartScreen.
After I allowed the execution on SmartScreen, it succeeds.
I think the validation is hanging in SmartScreen.
I will try to create a msi installer and check if it works, otherwise, I will cancel the PR

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2023-09-12 19:55:27.049   Exception message:                           
Internal error: Cannot install files to 64-bit locations on this version of Windows. 
2023-09-12 19:55:42.491   Exception message: 

image

**Comment by stephengillie** [comment]
`x86` validation is performed on a 32-bit OS. Is this a 64-bit application?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**